src/BatteryStateDisplayService.cpp

```diff
@@ -29,5 +30,13 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
         << " Current: " << batteryData.current
         << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
 
-    // TODO Print out time till it is depleted or charged.
+    if(batteryStateOfChargeService_.isCharging() == true)
+    {
+        QTextStream(stdout) <<"Time until battery is charged: " << batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString("hh:mm:ss.zzz") <<endl;
```

Comment:
There should be a space before and after all operators (+, -, *, <<, >, =, etc..)